### ISSUE TITLE
Update adapter in Repo ?!

### ISSUE THREAD

**Issue by Apollon77** [soef/ioBroker.lightify#12]
Hey,

Repo still has 0.0.28, NPM is at 0.2.7

**Comment by GermanBluefox** [comment]
No.

With soef@0.4.x I get always following error:
```
D:\pWork\node_modules\iobroker.lightify\node_modules\soef\soef.js:1322
    fns.adapter = fns.adapter(options);
                      ^

TypeError: fns.adapter is not a function
```

**Comment by GermanBluefox** [comment]
@soef Can you check it?

**Comment by Apollon77** [comment]
Saw that in Travis-CI too ... tests failing because of this

**Comment by soef** [comment]
Hi, i have fixed the problem in version 0.2.10 (soef 0.4.6). Older versions of utils.js returns Adapter, newer versions Adapter with a capital A letter.
